### PR TITLE
feat: analytics on toolbar

### PR DESCRIPTION
**Pull request by nicholasyao974**
# Description

copilot:summary

[- Link to Basecamp Todo](https://3.basecamp.com/3105655/buckets/35391220/todos/6813114390)

# How should this PR be QA Tested?

Please describe the QA tests that you ran to verify your changes. Please also note any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

# Walkthrough

copilot:walkthrough

### CODE REVIEW

File: apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+export function Analytics({ journey, variant }: AnalyticsProps): ReactElement {
+  return (
+    <NextLink
+      href={`/journeys/${journey.id}/reports`}
+      passHref
+      legacyBehavior
+      prefetch={false}
+    >
+      {variant === 'button' ? (
+        <Tooltip title="Analytics" arrow>
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx:25]: Is this tool tip from somewhere?

File: apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+import { JourneyFields as Journey } from '../../../../../../__generated__/JourneyFields'
+import { MenuItem } from '../../../../MenuItem/MenuItem'
+
+interface AnalyticsProps {
+  variant: 'button' | 'list-item'
+  journey: Journey
+}
+
+export function Analytics({ journey, variant }: AnalyticsProps): ReactElement {
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx:16]: Can you bring this component up a level (same level as `DeleteBlock`

File: apps/journeys-admin/src/components/Editor/EditToolbar/EditToolbar.tsx [hunk 3]
```diff
@@ -28,6 +29,7 @@ export function EditToolbar(): ReactElement {
     >
       {journey != null && (
         <>
+          {journey != null && <Analytics journey={journey} variant="button" />}
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/EditToolbar/EditToolbar.tsx:32]: This check is already been done
```suggestion
          {<Analytics journey={journey} variant="button" />}
```

Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/EditToolbar/EditToolbar.tsx:32]: This button also needs to be hidden on mobile

File: apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.spec.tsx [hunk 4]
```diff
@@ -7,9 +7,9 @@ import { JourneyProvider } from '@core/journeys/ui/JourneyProvider'
 import { TeamProvider } from '../../../../Team/TeamProvider'
 import { defaultJourney } from '../../../data'
 
-import { ReportMenuItem } from './ReportMenuItem'
+import { Analytics } from './Analytics'
 
-describe('ReportMenuItem', () => {
+describe('Analytics', () => {
```
Comment (csiyang) [inline apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.spec.tsx:12]: also add a test for the icon button variant

**Review by csiyang — CHANGES_REQUESTED** [review]

File: apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx (hunk 1 above)
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx:25]: It's just the MUI tooltip component.

File: apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx (hunk 2 above)
Comment (nicholasyao974) [reply apps/journeys-admin/src/components/Editor/EditToolbar/Menu/Analytics/Analytics.tsx]: Done, I also moved the `DuplicateBlock` component to the same level

**Review by csiyang — COMMENTED** [review]
looks good, let me know when it passes QA then i'll approve

**Review by csiyang — APPROVED** [review]

**Pull request merged**